content/c-sharp/concepts/loops/loops.md

```diff
@@ -88,3 +88,60 @@ This example outputs the following:
 2
 3
 ```
+## Nested Loops
+
+A loop can be placed inside another loop; this concept is known as **nested loops**. For every iteration of the outer loop, all iterations of the inner loop are executed. 
+
+
```

Comment:
remove one line and give a single blank line

Reply:
fixed